eventmesh-runtime/conf/eventmesh.properties

```diff
@@ -90,6 +90,14 @@ eventMesh.registry.plugin.server-addr=127.0.0.1:8848
 eventMesh.registry.plugin.username=nacos
 eventMesh.registry.plugin.password=nacos
 
+# The TLS configuration of registry plugin consul
+# keyStoreInstanceType's value can refer to com.ecwid.consul.transport.TLSConfig.KeyStoreInstanceType
+#eventMesh.registry.consul.tls.keyStoreInstanceType=
+#eventMesh.registry.consul.tls.certificatePath=
+#eventMesh.registry.consul.tls.certificatePassword=
+#eventMesh.registry.consul.tls.keyStorePath=
+#eventMesh.registry.consul.tls.keyStorePassword=
```

Comment:
why comment out these code?

Reply:
Because these are unnecessary configurations, and empty string is actually considered an illegal configuration.
Listing them here is to make users know which configuration items are available.